src/besselj.jl

```diff
@@ -265,6 +295,17 @@ function besselj_debye(v, x)
     return coef * Uk_poly_Jn(p, v, p2, T)
 end
 
+function besselj_debye_large_order(v, x)
+    T = eltype(x)
+    α = asech(x/v)
+    coef = exp(v*(tanh(α) - α))
```

Comment:
if x is large, (greater than 10) `tanh(α) - α` can just be `muladd(2, exp(-2x), -x)`